### PR TITLE
Add documentation on how to build and run the project

### PR DESCRIPTION
**Pull request by niccokunzmann**
This fixes https://github.com/ultrabug/mkdocs-static-i18n/issues/303

Hatch looks nice!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,6 +50,22 @@ But also in our hall of fame:
...
 Feel free to ask questions, enhancements and to contribute to this project!
 
+## Development
+
+The project is managed with `hatch`. [Install `hatch`](https://hatch.pypa.io/1.9/install/#gui-installer) first.
+
+Run the tests:
+
+```
+hatch test
```
Comment (ultrabug) [inline README.md:60]:
```suggestion
hatch run test:test
```

File: README.md [hunk 2]
```diff
@@ -50,6 +50,22 @@ But also in our hall of fame:
...
+```
+
+Serve the documentation:
+
+```
+hatch -e docs run mkdocs serve
```
Comment (ultrabug) [inline README.md:66]: `docs` is actually conflicting with the docs/ folder of the project which makes this command break, thanks for letting me see it :)

I'll open a PR to fix that first!

**Review by ultrabug — CHANGES_REQUESTED** [review]
You'd also maybe want to add how to run style checks

`hatch run style:check`

Thanks

**Comment by ultrabug** [comment]
FYI https://github.com/ultrabug/mkdocs-static-i18n/pull/308

File: README.md (hunk 2 above)
Comment (ultrabug) [inline README.md:66]:
```suggestion
hatch run doc:serve
```

File: README.md [hunk 3]
```diff
@@ -50,6 +50,23 @@ But also in our hall of fame:
...
+
+```
+hatch run doc:serve
+```
+
 ## Credits
 
-- Logo by [max.icons](https://www.flaticon.com/authors/maxicons)
\ No newline at end of file
+- Logo by [max.icons](https://www.flaticon.com/authors/maxicons)
```
Comment (ultrabug) [inline README.md:72]: what did you chance on this line, maybe you added a new line?

please try making sure it does not show in the diff

**Comment by niccokunzmann** [comment]
I added it to the documentation.

**Comment by niccokunzmann** [comment]
I added it to the documentation.

File: README.md (hunk 3 above)
Comment (niccokunzmann) [reply README.md:72]: I think, this might be inessential to the diff.
My editor probably added a line at the end.

Comment (ultrabug) [reply README.md:72]: Can you remove it then? this change should not be present in the diff

**Comment by ultrabug** [comment]
Thanks mate, moving on

**Pull request merged**